### PR TITLE
Add post order traversal

### PR DESCRIPTION
**Pull request by muthuannamalai12**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Added Post Order Traversal in C
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1923 
(Write your answer here.)

### CODE REVIEW

**Comment by muthuannamalai12** [comment]
@srimani-programmer . Kindly Review

File: C/ds/Postorder.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Post Order Traversal
```
Comment (atarax665) [inline C/ds/Postorder.c]: Add a beginner introduction of ```Post Order Traversal```.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code according to [C style guide](https://github.com/TesseractCoding/NeoAlgo/blob/master/C/C_Style_guide.md).

File: C/ds/Postorder.c (hunk 1 above)
Comment (muthuannamalai12) [reply C/ds/Postorder.c:1]: I have changed it. Kindly review it again @atarax665

**Comment by muthuannamalai12** [comment]
I have changed it Kindly review it again @atarax665

File: C/ds/Postorder.c [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	return tree;
+}
+
+void postorder(node *t)
+{
+	if (t != NULL)
+	{
+		postorder(t->left);
+		postorder(t->right);
+		printf(" %d", t->data);
```
Comment (atarax665) [inline C/ds/Postorder.c]:
```suggestion
		printf("%d ", t->data);
```

File: C/ds/Postorder.c [hunk 3]
```diff
@@ -0,0 +1,78 @@
+// Post Order Traversal
```
Comment (atarax665) [inline C/ds/Postorder.c]: Use multi-line comment.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by muthuannamalai12** [comment]
I have changed it. Kindly review it again @atarax665

**Review by atarax665 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

**Comment by muthuannamalai12** [comment]
I have solved the conflicts present in the branch. Kindly merge my Pull request @iamrajiv

**Pull request merged**